addon/components/o-s-s/nav-tab.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="fx-row fx-1 tab-container">
+  {{#each @tabArray as |tab|}}
+    <div class="fx-col tab {{if tab.selected "tab--selected"}} {{if tab.disabled "tab--disabled"}}"
+         {{on "click" this.onSelectTab}} type="button" tabindex="0">
```

Comment:
i think you have to pass the clicked tab as an argument here using: `{{on "click" (fn this.onSelectTab tab)}}`

The `fn` helper creates as function that wraps another one (the first parameter) and pass it the remaining parameters. This way you can have:
```
onSelectTab(selectedTab) {

}
```